requirements/base.in

```diff
@@ -20,7 +20,7 @@ repoze.lru
 requests
 requests-hawk
 sentry-sdk[flask]
-setuptools
+setuptools<71.0.1   # Remove this constraint once https://github.com/pypa/setuptools/issues/4482 is fixed
```

Comment:
This is the only dependency I had to pin. It cause this issue[1]:


```
Installed /usr/local/lib/python3.11/site-packages/balrog-3.53-py3.11.egg
Processing dependencies for balrog==3.53
error: [Errno 2] No such file or directory: '/usr/local/lib/python3.11/site-packages/setuptools/_vendor/importlib_resources-6.4.0.dist-info/METADATA'
error building image: error building stage: failed to execute command: waiting for process to exit: exit status 1
```

The rest went smoothly.

[1] https://firefox-ci-tc.services.mozilla.com/tasks/AlIPp-Y4RpqsNxJ-tK1GGQ/runs/1/logs/public/logs/live.log#L2243